### DISCUSSION TITLE
Сделайте наконец нормальный экспорт в HTML! / Finally make a normal export to HTML!

### DISCUSSION THREAD

**Discussion by tvorimvsevmeste** [freeplane/freeplane#2063] [Ideas]
Нужен нормальный экспорт в HTML !
Конечным итогом работы с таким софтом ВСЕГДА будет презентация и печатный отчёт. Печатный отчёт — это структурированный текст с рисунками. Для его формирования нужен экспорт в HTML / docx, где 
1. всё отображалось бы с таким форматированием, как на экране редактирования и без дополнительных танцев с бубном.
2. отступы соответствовали глубине вложенности
3. картинки имели бы подписи и нумерацию, а их размер можно было бы изменять.

Need normal export to HTML!
The end result of working with such software will ALWAYS be a presentation and a printed report. A printed report is a structured text with pictures. To generate it, you need export to HTML / docx, where
1. everything would be displayed with the same formatting as on the editing screen and without additional dancing with a tambourine.
2. the indents would correspond to the nesting depth
3. the pictures would have signatures and numbers, and their size could be changed.

**Comment by macmarrum** [comment]
There are a lot of export options in Freeplane. I think each user might have different expectations even for the same option, like HTML.

If you are after a portable and printable near-to-the-original representation of your mind map, you might want to try `Export Map...` -> `Scalable Vector Graphic (SVG)`.

As far as I know, other options might or might not represent the mind map as one can see it in Freeplane.
